packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -241,3 +247,310 @@ export class Load extends Op {
     stack.push(this.interpreter.scratch[this.index]);
   }
 }
+
+// err opcode : Error. Panic immediately.
+export class Err extends Op {
+  execute (stack: TEALStack): void {
+    throw new TealError(ERRORS.TEAL.TEAL_ENCOUNTERED_ERR);
+  }
+}
+
+// SHA256 hash of value X, yields [32]byte
+export class Sha256 extends Op {
+  execute (stack: TEALStack): void {
+    const hash = sha256.create();
+    const val = this.assertBytes(stack.pop()) as Message;
+    hash.update(val);
+    const hashedOutput = Buffer.from(hash.hex(), 'hex');
+    var arrByte = Uint8Array.from(hashedOutput);
+    stack.push(arrByte);
+  }
+}
+
+// SHA512_256 hash of value X, yields [32]byte
+export class Sha512_256 extends Op {
+  execute (stack: TEALStack): void {
+    const hash = sha512_256.create();
+    const val = this.assertBytes(stack.pop()) as Message;
+    hash.update(val);
+    const hashedOutput = Buffer.from(hash.hex(), 'hex');
+    var arrByte = Uint8Array.from(hashedOutput);
+    stack.push(arrByte);
+  }
+}
+
+// If A < B pushes '1' else '0'
+export class LessThan extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a < b) {
+      stack.push(BigInt('1'));
```

Comment:
Create two constants: `BIGINT0`, `BIGINT1` and use everywhere (for below opcodes as well). Or maybe `TRUE` = `BigInt('1')`, ..

Reply:
I think BIGINT0 is better

Reply:
:+1: